analyzers/GreyNoise/greynoisev3.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from collections import defaultdict, OrderedDict
+
+from cortexutils.analyzer import Analyzer
+from greynoise import GreyNoise
+import requests
+
+
+class GreyNoiseAnalyzer(Analyzer):
+    """
+    GreyNoise API docs: https://developer.greynoise.io/reference#noisecontextip-1
+    """
+
+    def run(self):
+
+        if self.data_type == "ip":
+            api_key = self.get_param('config.key', None)
+            api_client = GreyNoise(api_key=api_key, timeout=30)
```

Comment:
@markus-nclose : can we also get an integration name passed here?

Example:
api_client = GreyNoise(api_key=api_key, integration_name="greynoise-cortex-analyzer-v3.0")

We're working to set this up as our standard going forward.

Reply:
Added as requested above, looks like the only use is in the user-agent header?

Reply:
Thanks, yes that is correct, this is essentially a user-agent string for us to see on the backend.  It helps us understand the adoption and use of integrations with our service and we can also proactively let users know there is a new version available if we see high usage of older versions.